plugins/scm.js

```diff
@@ -72,6 +72,14 @@ const GET_COMMIT_SHA = Joi.object().keys({
     scmRepo: Scm.repo.optional()
 }).required();
 
+const ADD_PR_COMMENT = Joi.object().keys({
+    scmUri,
+    token,
+    prNum,
```

Comment:
this is optional when it's defined: https://github.com/screwdriver-cd/data-schema/blob/8ba40a1be92babc54d73bedb5aea5a6cad932b51/plugins/scm.js#L13

is it the intention here to make it optional?

Reply:
no, I'll change it to make it required